flow.md

```diff
@@ -0,0 +1,47 @@
+# Flow
+
+## Git flow
+
+- default branch is **develop**
+   - Consists of commits where each commit represents fully completed and reviewed task
+   - Contains tags for releases
+   - History should never be rewritten
+- 1 task = 1 branch = 1 pull request - 1 commit to **develop**
+   - mark bug tasks as bugfix/
+   - mark nornal tasks as feature/
+- Merge requires 2 approval in pull request
+- Pull request consists of
+   - Task tag
+   - Description
+   - Affected parts of the project
+
+> **Commit** in **develop** example:
+> BS-01 Improve SomeService background fetch
+
+> **Branch** example:
+> feature/BS-01-some-service-bg-fetch
```

Comment:
maybe it will be better if we leave only task tag in branch name

Reply:
I think it is not really necessary because some task description here might save us a time to not jump into task tracker and find task for the tag

Reply:
ok agree here